spec/journal_title_spec.rb

```diff
@@ -8,7 +8,7 @@
     it "exact title matches should be first" do
       orig_query_str = solr_params['q'].split('}').last
       resp = solr_resp_ids_titles(solr_params)
-      expect(resp).to include({'title_245a_display' => /^#{orig_query_str}\W*$/i}).in_each_of_first(exp_ids.size)
+      expect(resp).to include({'title_245a_display' => /^#{orig_query_str}.*$/i}).in_each_of_first(exp_ids.size)
```

Comment:
this is no longer "exact title";  `\W` means NOT a-z, A-Z, 0-9, or _ (underscore)  Another one for pairing?